src/index.ts

```diff
@@ -101,7 +101,16 @@ async function install(client: InstallableClient, versionRange: semver.Range): P
         + `resolved successfully to ${JSON.stringify(clientInfo)}`);
 
     let executablePath: string;
-    const executablePathFromCache = await retreiveFromCache(clientInfo);
+    let executablePathFromCache;
+
+    if (isGhes()) {
+        ghCore.info(`⏩ Skipping cache as it is not supported on GitHub Enterprise Server. `
```

Comment:
I do not like how the 'skipping cache' emoji is the same as the 'downloading from cache' emoji, can you use something else and also use it in the skipping cache message below

Reply:
'GitHub enterprise detected; skipping cache. For more information, see <issue>'

Reply:
should this logic be moved into `retrieveFromCache` where the existing env var is checked?